code/modules/medical/genetics/geneticsMachines.dm

```diff
@@ -561,7 +561,7 @@
 			if (addEffect) // re-mutantify if we would have been able to anyway
 				subject.bioHolder.AddEffect(addEffect)
 			if (genResearch.emitter_radiation > 0)
-				subject.changeStatus("radiation", (genResearch.emitter_radiation) SECONDS, 3)
+				subject.take_radiation_dose(0.5 SIEVERTS)
```

Comment:
this should take `genResearch.emitter_radiation` into account (probably have to tweak the numbers on that var to make sense with new rads), since that's how the "radiation dampeners" upgrades do